docs/structured-streaming-programming-guide.md

```diff
@@ -487,11 +487,11 @@ Dataset[Row] csvDF = spark
 spark = SparkSession. ...
 
 # Read text from socket 
-socketDF = spark \
-    .readStream()  \
-    .format("socket") \
-    .option("host", "localhost") \
-    .option("port", 9999) \
+socketDF = spark\
```

Comment:
@srowen, there is no clear definition of it.

https://docs.python.org/2/reference/lexical_analysis.html#explicit-line-joining
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

I use:
```python
# with space
if 1900 < year < 2100 and 1 <= month <= 12:
    return 1

if 1900 < year < 2100 \
    and 1 <= month <= 12:
    return 1

# without space
s = 'Hello world'.lower().split()

s = 'Hello world'.lower()\
    .split()
```

This makes the code more readable.